src/Components/Topic.jsx

```diff
@@ -102,6 +108,24 @@ function Topic() {
         }
     }
     
+    if(invalidTopic !== "none") {
+        return <div key={topic_name_with_emoji} className="topics">
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+
+       <h1>No such Topic...</h1>
+        <div className="invalidTopicContainer">
+             <Link to="/topics"><button className="topicButton">Click to see Topics to choose from</button></Link>
+        </div>
+        <br></br>
+        <br></br>
+     
```

Comment:
There's a lot of similarities with this error and your article error. Could there be an Error component that does the job for both? Worth a look